components/brave_new_tab_ui/actions/new_tab_actions.ts

```diff
@@ -32,6 +32,10 @@ export const dismissBrandedWallpaperNotification = (isUserAction: boolean) =>
 export const preferencesUpdated = (preferences: Preferences) =>
   action(types.NEW_TAB_PREFERENCES_UPDATED, preferences)
 
+export const updateClockWidget = (showClockWidget: boolean,
```

Comment:
nit: this action name is unclear where this "update" came from. If it was `clockSettingsUpdated` I would assume that it's purpose is to relay settings changes to the store. If it's `toggleClockFormat` then I would assume it's a user action that we should relay to the API.  [Here's a guide on redux action names that I've found helpful.](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters)

Reply:
reviewFeedbackImplemented() 😄 👍